modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -107,41 +116,40 @@ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b75
 
 ----
 
-
 .. Ensure that the logging collector Pods are using a 4.3 image:
 +
 ----
 $ oc get pod -n openshift-logging --selector logging-infra=fluentd -o yaml |grep 'image:'
 
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.3.0-202001081344
 ----
 
 .. Ensure that the Kibana Pods are using a 4.3 image:
 +
 ----
 $ oc get pod -n openshift-logging --selector logging-infra=kibana -o yaml |grep 'image:'
 
-image: registry.redhat.io/openshift4/ose-logging-kibana5:v4.3.0-201909210748
-image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.3.0-201909201915
-image: registry.redhat.io/openshift4/ose-logging-kibana5:v4.3.0-201909210748
-image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.3.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-kibana5:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/oauth-proxy:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-kibana5:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/oauth-proxy:v4.3.0-202001081344
```

Comment:
It should be registry.redhat.io/openshift4/ose-oauth-proxy:v4.3.0-202001081344

Reply:
@anpingli All the images I have as `oauth-proxy` should be `ose-oauth-proxy`? I made that change.

Reply:
Yes, it should have the prefix ose-.